docs/config.md

```diff
@@ -2,3 +2,71 @@
 Set the scenarios to inject and the tunings like duration to wait between each scenario in the config file located at [config/config.yaml](https://github.com/redhat-chaos/krkn/blob/main/config/config.yaml).
 
 **NOTE**: [config](https://github.com/redhat-chaos/krkn/blob/main/config/config_performance.yaml) can be used if leveraging the [automated way](https://github.com/redhat-chaos/krkn#setting-up-infrastructure-dependencies) to install the infrastructure pieces.
+
+Config components: 
+* [Kraken](#kraken)
+* [Cerberus](#cerberus)
+* [Performance Monitoring](#performance-monitoring)
+* [Tunings](#tunings)
+
+# Kraken 
+This section defines scenarios and specific data to the chaos run 
+
+## Distribution
+Either **openshift** or **kubernetes** depending on the type of cluster you want to run chaos on. 
+The prometheus url/route and bearer token are automatically obtained in case of OpenShift, please set it when the distribution is Kubernetes.
+
+## Exit on failure
+**exit_on_failure**:  Exit when a post action check or cerberus run fails
+
+## Publish kraken status
+**publish_kraken_status**: Can be accessed at http://0.0.0.0:8081 (or what signal_address and port you set in signal address section)
+**signal_state**: State you want kraken to start at; will wait for the RUN signal to start running a chaos iteration. When set to PAUSE before running the scenarios, refer to [signal.md](signal.md) for more details
+
+## Signal Address 
+**signal_address**: Address to listen/post the signal state to
+**port**: port to listen/post the signal state to
+
+## Litmus Variables
+Litmus installation specifics if you are running one of the hog scenarios. See [litmus doc](litmus_scenarios.md) for more informaiton on these types of scenarios 
+**litmus_install**: Installs specified version of litmus, set to False if it's already setup
+**litmus_version**: Litmus version to install
+**litmus_uninstall**: If you want to uninstall litmus if failure
+**litmus_uninstall_before_run**: If you want to uninstall litmus before a new run starts, True or False values
+
+
+## Chaos Scenarios 
+
+**chaos_scenarios**: List of different types of chaos scenarios you want to run with paths to their specific yaml file configurations
```

Comment:
@paigerube14 Maybe mention that the scenarios are currently run one after other and will exit if one of the scenarios fail without moving onto the next one. Thoughts?